packages/studio-ui/src/web/views/Nlu/entities/ListEntityOccurrence.tsx

```diff
@@ -24,7 +24,10 @@ export const Occurrence: FC<Props> = props => {
       }
       placeholder={lang.tr('nlu.entities.synonymPlaceholder')}
       rightElement={<Button icon="delete" minimal onClick={props.remove} />}
-      onChange={(synonyms: string[]) => props.onChange({ ...props.occurrence, synonyms })}
+      onChange={(synonyms: string[]) => {
+        synonyms = synonyms.map(s => s.trim())
```

Comment:
Why do we need to re-trim them on every change? Is this in order to fix older occurrences that weren't trimmed?